cray_product_catalog/catalog_delete.py

```diff
@@ -60,6 +60,8 @@
 LOGGER.setLevel(logging.INFO)
 handler = logging.StreamHandler(sys.stdout)
 handler.setLevel(logging.DEBUG)
+formatter = logging.Formatter("%(levelname)s %(message)s")
+handler.setFormatter(formatter)
```

Comment:
I'm curious why logger configuration is being done a per-module basis here. My understanding is that by convention, the logging configuration is applied to the root logger, and child loggers (e.g. module-level loggers instantiated by `logging.getLogger()`) inherit from that configuration. With the per-module solution, there is less flexibility for log formatting for clients, and there is duplication of the log format for each module. I understand that the logging was already set up that way, but I wonder if we can change it easily now as part of this change.

Reply:
Now that you mention it, another problem I think I see with this approach is that other modules under `cray_product_catalog` don't get the formatting applied (or work at all) e.g. log statements from the modules under `util/`

Reply:
I re-pushed the change if you want to take a look. Sorry, probably would have been easier to review if I'd used a separate commit to be squashed later.

> by convention, the logging configuration is applied to the root logger

I don't think I did *exactly* this in that I'm not applying configuration to the `root` logger but rather the top-level logger for this module, i.e. `cray_product_catalog`. I think that still solves the issues of (a) not duplicating the code and (b) fixing logging in child modules.

Reply:
I think the updated solution should be good, it looks more or less like what we're doing in SAT at least.